### ISSUE TITLE
Seeking -10 second actually goes back 3 seconds at a time

### ISSUE THREAD

**Issue by sn0wyfall** [iina/iina#3110]
**System and IINA version:**

- macOS 
<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**
Invoke the command `seek -10`

- [ ] MPV does not have this problem.

**How often does this happen?**
All the time! Has been happening for months.

**Comment by sterid** [comment]
Almost the same problem.
Do you have a pause for 2-5 sec when you are changing playhead position?

**Comment by MikeWang000000** [comment]
Same problem here.

I finally found a possible solution: Use `seek -10 exact` instead.
However, it performs a little slow.

https://mpv.io/manual/stable/#list-of-input-commands

    seek <target> [<flags>]
    ...
    keyframes
        Always restart playback at keyframe boundaries (fast).
    exact
        Always do exact/hr/precise seeks (slow).
    ...

I guess IINA uses `keyframes` flag by default for better performance.
But it isn't as accurate as I expected.

**Comment by sn0wyfall** [comment]
@MikeWang000000 That's for the tip! I'll try that instead. So is this a bug with IINA or mpv?

**Comment by chaimk** [comment]
I have even wilder inconsistencies, where pressing the same key to invoke `seek 7` results in jumping forward roughly 7 seconds the _first_ time you press it but then jumps 20-30 seconds for each subsequent keypress. And if I `seek 7` a few times then try to go backwards (`seek -7`), it jumps _**forward**_ again one more time. In addition, there is a deviation between the popup indicator value (top left) and the timeline indicator (bottom center). It's very strange and quite annoying.

Having said all this, using `seek 7 exact` as @MikeWang000000 suggested seems to fix this. Thank you! Also, at least on my MPB 15" 2019 (6-core) it works pretty smoothly.

It would be good if using `exact` were changed to the default mpv command, since without it it is very user-unfriendly.

**Comment by sn0wyfall** [comment]
@chaimk Yeah your description seems pretty accurate. I also have just strange behavior when using `seek` by itself
